coderd/database/migrations/000234_fix_notifications_user_created.up.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE notification_templates
+SET
+    body_template = E'Hi {{.UserName}},\n\nNew user account **{{.Labels.created_account_name}}** has been created.'
```

Comment:
This was incorrectly set in the [initial migration](https://github.com/coder/coder/blob/83963b9e610c58a99c8c516e8eeb00a37a428ed9/coderd/database/migrations/000233_notifications_user_created.up.sql#L4), let's sneak it in here if you don't mind

```suggestion
    body_template = E'Hi {{.UserName}},\n\nNew user account **{{.Labels.created_account_name}}** has been created.',
    group = 'User Events'
```

Reply:
I would rather keep this idempotent. It does not seem to be an urgent fix that requires modifying existing files, and if there are some scripts checking the content of migration files, and I wouldn't like to make them suffer.

Reply:
Maybe you missed the content of the change? The event is a user event, but its group is incorrectly set to `Workspace Events` currently.

Reply:
Oh crap, let me redo it.

Reply:
https://github.com/coder/coder/pull/14112